src/grading-settings/grading-sidebar/GradingSidebar.test.jsx

```diff
@@ -16,11 +19,31 @@ jest.mock('react-router-dom', () => ({
 
 const RootWrapper = () => (
   <IntlProvider locale="en" messages={{}}>
-    <GradingSidebar intl={injectIntl} courseId="123" />
+    <AppProvider store={store}>
+      <GradingSidebar intl={injectIntl} courseId="123" />
+    </AppProvider>
   </IntlProvider>
 );
 
 describe('<GradingSidebar />', () => {
+  beforeEach(async () => {
+    initializeMockApp({
+      authenticatedUser: {
+        userId: 3,
+        username: 'abc123',
+        administrator: false,
+        roles: [],
+      },
+    });
+
+    store = initializeStore({
+      courseDetail: {
+        courseId: 'id',
+        status: 'sucessful',
+      },
+    });
+  });
```

Comment:
```suggestion
  beforeEach(async () => {
    initializeMocks();
  });
```

There are nice helpers in [`testUtils`](https://github.com/openedx/frontend-app-authoring/blob/master/src/testUtils.tsx) that do this boilerplate for you. You can also remove `RootWrapper` above, as it's already provided if you use [the `render` function from `testUtils`](https://github.com/openedx/frontend-app-authoring/blob/cff1177ae93db0f63377ef387fca232b24e47c0b/src/testUtils.tsx#L195)